spec/features/ui/v1/digital_objects/metadata/edit_spec.rb

```diff
@@ -5,7 +5,7 @@
 RSpec.describe 'Digital Object Edit', solr: true, type: :feature, js: true do
   let(:uid) { SecureRandom.uuid }
   let(:item) { FactoryBot.create(:item, uid: uid) }
-  let(:permissons_required) { [] }
+  let(:permissions_required) { [] }
```

Comment:
don't be reluctant to pull this kind of change out and send it straight to `main` (if you don't mind the rebasing as you go)

Reply:
Done!  Good idea, thanks!